src/renderer/scss/_gui.scss

```diff
@@ -107,6 +114,7 @@ input {
 }
 
 input::placeholder {
+  color: var(--input-placeholder-color) !important;
```

Comment:
We shouldn't need an `!important` here. If there is something that is overriding this style, we should remove that, or make this a more specific rule with classnames